ci/matrix.yaml

```
@@ -54,11 +57,11 @@ workflows:
     # default_projects: nvcc
     - {jobs: ['build'], std: 'all', ctk: *ctk_11_1, cxx: [*gcc6, *gcc7, *gcc8, *gcc9, *llvm9, *msvc2017]}
     - {jobs: ['build'], std: 'all', ctk: *ctk_11_8, cxx: [*gcc11], sm: '60;70;80;90'}
-    - {jobs: ['build'], std: 'all', cxx: [*gcc7, *gcc8, *gcc9, *gcc10, *gcc11]}
-    - {jobs: ['build'], std: 'all', cxx: [*llvm9, *llvm10, *llvm11, *llvm12, *llvm13, *llvm14, *llvm15]}
-    - {jobs: ['test'],  std: 'all', cxx: [*gcc12, *llvm16]}
-    - {jobs: ['build'], std: 'all', cxx: [*gcc12, *llvm16], cpu: 'arm64'}
-    - {jobs: ['build'], std: 'all', cxx: [*gcc12], sm: '90a'}
+    - {jobs: ['build'], std: 'all', cxx: [*gcc7, *gcc8, *gcc9, *gcc10, *gcc11, *gcc12]}
+    - {jobs: ['build'], std: 'all', cxx: [*llvm9, *llvm10, *llvm11, *llvm12, *llvm13, *llvm14, *llvm15, *llvm16, *llvm17, *llvm18]}
```

```suggestion
    - {jobs: ['build'], std: 'all', cxx: [*llvm9, *llvm10, *llvm11, *llvm12, *llvm13, *llvm14, *llvm15, *llvm16, *llvm17]}
```

The `test` job for `llvm-newest` below will automatically generate a `build` job for `llvm18`. Including it here would will generate two build jobs.

The build-workflow script *should* catch this, remove the duplicate and warn, but it looks like that functionality has a bug. PR incoming.

---

https://github.com/NVIDIA/cccl/pull/1759